src/test/java/com/wikia/webdriver/testcases/adstests/TestVUAP.java

```diff
@@ -11,16 +12,41 @@
   private static final Dimension DESKTOP_SIZE = new Dimension(1920, 1080);
 
   @Test(
-          groups = "AdsUapDesktop"
```

Comment:
could you add a unique group to each test so it's possible to run it on jenkins/command line?